packages/cryptoplease_link/lib/src/constants.dart

```diff
@@ -16,6 +16,7 @@ const moonpayHost = 'moonpay.$cpLinkDomain';
 const shareableLinkPaymentFee = 100000; // 0.1 USDC
 const directPaymentFee = 10000; // 0.01 USDC
 const directPaymentWithAccountCreationFee = 100000; // 0.1 USDC
+const swapFee = 100000; // 0.1 USDC
```

Comment:
`swapFee` is not a constant, no? I think the outcome of the discussion was that we have to do dynamic fees. Or maybe I'm confused it with something.

Reply:
Should be fixed by https://github.com/cryptoplease/cryptoplease-dart/pull/638